pkg/tenant/members.go

```diff
@@ -123,6 +123,12 @@ func (s *Server) MemberCreate(c *gin.Context) {
 		return
 	}
 
+	// Verify the role provided is valid.
```

Comment:
This check was included with `MemberRoleUpdate`,  so I added it here in `MemberCreate` and `MemberUpdate` while updating the handlers and tests.